.eslintrc.cjs

```diff
@@ -100,6 +102,10 @@ module.exports = {
 		"simple-import-sort/imports": "error",
 
 		// These on-by-default rules don't work well for this repo and we like them off.
+		"jsdoc/require-jsdoc": "off",
+		"jsdoc/require-param": "off",
+		"jsdoc/require-property": "off",
+		"jsdoc/require-returns": "off",
```

Comment:
The `jsdoc` plugin was declared in an `overrides` for `**/*.ts` files, so it overrides any root-level rule configurations. I'll move these to the overrides.

fyi @emday4prez

Reply:
f593225